.github/workflows/release-provider.yml

```
@@ -46,10 +46,10 @@ jobs:
             ${{ runner.os }}-go-
       - name: Import GPG key
         id: import_gpg
-        uses: crazy-max/ghaction-import-gpg@v2
-        env:
-          GPG_PRIVATE_KEY: ${{ secrets.GPG_PRIVATE_KEY }}
-          PASSPHRASE: ${{ secrets.PASSPHRASE }}
+        uses: crazy-max/ghaction-import-gpg@v3
+        with:
+          gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
```

😎 
```suggestion
          gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }}
```

https://github.com/crazy-max/ghaction-import-gpg/blob/master/CHANGELOG.md#300-20200906

> - Move `GPG_PRIVATE_KEY` env var to `gpg-private-key` input
> - Move `PASSPHRASE` env var to `passphrase` input

---

Oh, nice catch, Thank you 🙇